examples/scripts/build-android

```diff
@@ -79,7 +79,7 @@ while [[ $# -gt 0 ]]; do
     esac
 done
 
-if [ $BUILD_LOCAL_GLUECODIUM ]; then
+if $BUILD_LOCAL_GLUECODIUM; then
```

Comment:
This will execute whatever is in the variable and branch based on the exit status. I think it would be safer to compare the variable contents instead.
`if [ "$BUILD_LOCAL_GLUECODIUM" = "true" ]; then ...`

Reply:
Done